tests/conftest.py

```diff
@@ -44,3 +44,8 @@ def vouched_user(stored_users):
 @pytest.fixture
 def private_user(stored_users):
     return stored_users['private']
+
+
+@pytest.fixture
+def unvouched_user(stored_users):
```

Comment:
That's a much better name :). Would it be better to reuse the "new_user" that already exists? That would basically require going through the who registration process as in test_register. For now I'm not choosing to do that but let me know if that would be preferred.